api/envoy/extensions/transport_sockets/quic/v3/quic_transport.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+
+package envoy.extensions.transport_sockets.quic.v3;
+
+import "envoy/extensions/transport_sockets/tls/v3/tls.proto";
+
+import "udpa/annotations/status.proto";
+import "udpa/annotations/versioning.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.transport_sockets.quic.v3";
+option java_outer_classname = "QuicTransportProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: quic transport]
+// [#extension: envoy.transport_sockets.quic]
+
+// Configuration for QUIC transport socket. This provides Google's QUIC implementation to Envoy.
```

Comment:
for QUIC -> for Downstream QUIC (and upstream below)

"This provides" doesn't seem quite right.  Also is this just for Google-QUIC or also used for IETF-QUIC, just Google's implementation?